src/main/app/claims/models/claim.ts

```diff
@@ -93,7 +93,7 @@ export class Claim {
       const interestDate: InterestDate = this.claimData.interestDate
       const fromDate: Moment = interestDate.type === InterestDateType.SUBMISSION ? this.createdAt : interestDate.date
 
-      interest = calculateInterest(claimAmount, interestRate, fromDate, toDate)
+      interest = await calculateInterest(claimAmount, interestRate, fromDate, toDate)
```

Comment:
I don't think await is needed here, this is an async method and you aren't doing anything after it,
REC